packages/theme-default/src/logic/utils.ts

```diff
@@ -51,3 +51,10 @@ export function renderInlineMarkdown(text: string) {
     .replace(/`(.*?)`/g, '<code>$1</code>');
   return renderHtmlOrText(htmlText);
 }
+
+export function parseInlineMarkdownText(mdx: string) {
+  return mdx
```

Comment:
We can use const to store these regular expressions, so that they are easier to maintain later, including the above `renderInlineMarkdown` functions.

By the way, I noticed in former PR, `renderInlineMarkdown` functions has been modified but the param comment not updated, you can updated it in this PR